### PR TITLE
Stripe customer exists without card

### PR DESCRIPTION
**Pull request by petedejoy**
This PR handles the edge case in which a customer exists in Stripe _without_ an associated credit card. This might happen if we sign an annual deal with a Cloud customer and invoice them for the year up-front rather than requiring a credit card for monthly payment.

https://github.com/astronomer/issues/issues/400

### CODE REVIEW

File: src/modules/workspaces/ConfigureBilling/UpdateBilling.js [hunk 1]
```diff
@@ -16,51 +16,56 @@ import s from './styles.scss'
...
-          </div>
-        </Box>
+        {card ? (
+          <Box>
+            <H3>Current Card on File</H3>
+            <div className={s.cardPadding}>
+              <PaymentCard
+                bank="default"
+                brand={cardBrand}
+                number={cardNumber}
```
Comment (schnie) [inline src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: Could we drop `const cardNumber` up above and use `card.last4` here? Just to simplify.

File: src/modules/workspaces/ConfigureBilling/UpdateBilling.js [hunk 2]
```diff
@@ -16,51 +16,56 @@ import s from './styles.scss'
 class UpdateBilling extends React.Component {
   render() {
     const { form, card } = this.props
-    const cardNumber = '************' + card.last4
-    const expirationDate = card.expMonth + '/' + card.expYear
-    const cardBrand = card.brand.toLowerCase() //Stripe returns with a capital first letter adn the component requires a lowercase string
+    const cardNumber = card ? '************' + card.last4 : null
+    const expirationDate = card ? card.expMonth + '/' + card.expYear : null
```
Comment (schnie) [inline src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: Should we default to `null` or empty string here? Prob doesn't make a difference but a `string` like `""` might make slightly more sense.

File: src/modules/workspaces/ConfigureBilling/UpdateBilling.js [hunk 3]
```diff
@@ -16,51 +16,56 @@ import s from './styles.scss'
...
+    const cardBrand = card ? card.brand.toLowerCase() : null //Stripe returns with a capital first letter adn the component requires a lowercase string
```
Comment (schnie) [inline src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: Same here

**Comment by schnie** [comment]
@petedejoy also looks like there's a lint issue.

File: src/modules/workspaces/ConfigureBilling/UpdateBilling.js (hunk 1 above)
Comment (petedejoy) [reply src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: ✅

**Comment by petedejoy** [comment]
@schnie - just added all the changes you suggested here and tested to make sure everything still looks good. Hoping I pass the linter this time...

File: src/modules/workspaces/ConfigureBilling/UpdateBilling.js (hunk 1 above)
Comment (petedejoy) [reply src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: ✅

File: src/modules/workspaces/ConfigureBilling/UpdateBilling.js (hunk 2 above)
Comment (petedejoy) [reply src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: ✅

File: src/modules/workspaces/ConfigureBilling/UpdateBilling.js (hunk 3 above)
Comment (petedejoy) [reply src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: ✅

File: src/modules/workspaces/ConfigureBilling/UpdateBilling.js (hunk 1 above)
Comment (schnie) [reply src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: Oh damnit. I saw that you were doing some formatting with the prefix. Maybe the way you had it was better. My fault!

Comment (petedejoy) [reply src/modules/workspaces/ConfigureBilling/UpdateBilling.js]: Ha all good! Just reverted back on this.

**Pull request merged**